charts/argo-cd/templates/argocd-configs/repository-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- range $repo_key, $repo_value := .Values.configs.repositories }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-repo-{{ $repo_key }}
+  labels:
+    argocd.argoproj.io/secret-type: repository
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.server.name "name" "secret") | nindent 4 }}
```

Comment:
I think we can omit `"component" $.Values.server.name "name" "secret"` as this secret is not targeted to one component of ArgoCD.